internal/cli/deployment_destroy.go

```diff
@@ -75,10 +78,13 @@ func (c *DeploymentDestroyCommand) Run(args []string) int {
 					Deployment: deployment,
 				},
 			}); err != nil {
-				c.ui.Output("Error destroying the deployment: %s", err.Error(), terminal.WithErrorStyle())
-				return ErrSentinel
+				c.ui.Output("Error destroying deployment %d: %s", deployment.Sequence, err.Error(), terminal.WithErrorStyle())
+				destroymentErrors = append(destroymentErrors, err)
 			}
 		}
+		if len(destroymentErrors) > 0 {
+			return errors.New("one or more deployments failed to be destroyed")
```

Comment:
This is the CLI so I think we should be returning an exit code and printing the error to the user, right? 🤔